client/src/components/generic/ExecImage/ExecImage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react"
+import Image from "next/image"
+import "./ExecImage.css"
+
+export interface ExecImageProps {
+  src: string
+  alt: string
+  title: string
+  name: string
+}
+
+const ExecImage: React.FC<ExecImageProps> = ({ src, alt, title, name }) => {
+  const [isHovered, setIsHovered] = useState(false)
+
+  const handleTouchStart = () => {
+    setIsHovered(true)
+  }
+
+  const handleTouchEnd = () => {
+    setIsHovered(false)
+  }
```

Comment:
https://tailwindcss.com/docs/hover-focus-and-other-states

Reply:
Even though it is super easy, we should avoid javascript for styling whenever possible:

- CSS can take advantage of GPU acceleration more often compared to JS
- Javascript increases styling complexity and can lead to unnecessary re-renders